### PR TITLE
Remove undocumented Windows APIs (NtCreateKeyedEvent, NtReleaseKeyedEvent, NtWaitForKeyedEvent) in OpenJDK.

### PR DESCRIPTION
**Pull request by dhanalla**
Backport of https://github.com/openjdk/jdk/pull/20682 to JDK17

### CODE REVIEW

File: ms-patches/remove-undocumentedAPI.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title: Remove undocumented windows APIs
```
Comment (brianjstafford) [inline ms-patches/remove-undocumentedAPI.yaml:1]: Please capitalize Windows

**Review by brianjstafford — APPROVED** [review]
LGTM - just fix the capitalization of Windows

File: ms-patches/remove-undocumentedAPI.yaml (hunk 1 above)
Comment (dhanalla) [reply ms-patches/remove-undocumentedAPI.yaml]: Done.

**Comment by dhanalla** [comment]
> LGTM - just fix the capitalization of Windows

Thanks @brianjstafford.

**Pull request merged**